packages/core/upload/admin/src/components/EditAssetDialog/PreviewBox/index.js

```diff
@@ -104,13 +111,12 @@ export const PreviewBox = ({
       trackUsage('didCropFile', { duplicatedFile: false, location: trackedLocation });
     }
 
-    stopCropping();
-    onCropCancel();
     setAssetUrl(optimizedCachingImage);
     setThumbnailUrl(optimizedCachingThumbnailImage);
+    setHasCropIntent(false);
   };
 
-  const isInCroppingMode = isCropping && !isLoading;
+  const isInCroppingMode = asset.isLocal ? true : isCropping && !isLoading;
```

Comment:
I think this change of condition is causing an UI issue 🤔
`ActionRow` takes the `blurry` props to define the background color and it's always true now:

<img width="861" alt="image" src="https://user-images.githubusercontent.com/71838159/177574976-61bcd2bf-3f51-4537-9755-b61611212af2.png">

Reply:
I think I am not able to reproduce the issue you have mentioned:

https://user-images.githubusercontent.com/2244375/178478524-1a01d17d-d6f4-4b25-9675-67e8111f685a.mp4

Do I do something wrong?

Reply:
Thank you again for the thourough testing! Glad to see you found this bug, which should be fixed by 7c15d41ace17e97dbe8819a2e08eea768902d95a